File: app/src/main/java/App/repositories/retrofit/responses/PersonasResponse.java
package App.repositories.retrofit.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import App.repositories.room.entities.Personas;

public class PersonasResponse {



    @SerializedName("status")
    @Expose
    private String status;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("data")
    @Expose
    private List<Personas> personas = null;

    public PersonasResponse(String status, String message, List<Personas> personas) {
        this.status = status;
        this.message = message;
        this.personas = personas;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Personas> getPersonas() {
        return personas;
    }

    public void setPersonas(List<Personas> personas) {
        this.personas = personas;
    }
    
}

File: app/src/main/java/App/repositories/retrofit/responses/PersonasResponse.kt
package App.repositories.retrofit.responses

import App.repositories.room.entities.Personas
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PersonasResponse(@field:Expose @field:SerializedName("status") var status: String, @field:Expose @field:SerializedName("message") var message: String, personas: List<Personas>?) {

    @SerializedName("data")
    @Expose
    var personas: List<Personas>? = null

    init {
        this.personas = personas
    }
}